### PR TITLE
Tech 801 raise error when expected node not found

### PR DESCRIPTION
**Pull request by cpartin**

### CODE REVIEW

**Comment by jedprentice** [comment]
@cpartin how did you test this?

**Comment by cpartin** [comment]
Pretty much impossible to write a spec for this because of fakeweb, at least without breaking all the others.

**Comment by jedprentice** [comment]
OK, I guess we'll have to just keep an eye on it when we deploy it.

File: .rvmrc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # development environment upon cd'ing into the directory
 
 # First we specify our desired <ruby>[@<gemset>], the @gemset name is optional.
-environment_id="ruby-1.9.2-p290"
+environment_id="ruby-2.0.0-p451"
```
Comment (awwaiid) [inline .rvmrc]: If you're going to mess with this, just rm it and put in a .ruby-version (which is the common standard)

Comment (awwaiid) [inline .rvmrc]: Oh already merged. nevermind.

**Pull request merged**